src/sentry/api/endpoints/organization_dashboards.py

```diff
@@ -43,28 +57,35 @@ def has_object_permission(self, request: Request, view, obj):
         return True
 
 
+@extend_schema(tags=["Dashboards"])
 @region_silo_endpoint
 class OrganizationDashboardsEndpoint(OrganizationEndpoint):
     publish_status = {
-        "GET": ApiPublishStatus.UNKNOWN,
-        "POST": ApiPublishStatus.UNKNOWN,
+        "GET": ApiPublishStatus.PUBLIC,
+        "POST": ApiPublishStatus.PUBLIC,
     }
     owner = ApiOwner.PERFORMANCE
     permission_classes = (OrganizationDashboardsPermission,)
 
+    @extend_schema(
+        operation_id="Retrieve an Organization's Dashboards",
+        parameters=[GlobalParams.ORG_ID_OR_SLUG],
+        request=None,
+        responses={
+            200: inline_sentry_response_serializer(
+                "DashboardListResponse", list[DashboardListResponse]
+            ),
+            400: RESPONSE_BAD_REQUEST,
+            403: RESPONSE_FORBIDDEN,
+            404: RESPONSE_NOT_FOUND,
+        },
+        examples=DashboardExamples.DASHBOARDS_QUERY_RESPONSE,
+    )
     def get(self, request: Request, organization) -> Response:
         """
-        Retrieve an Organization's Dashboards
-        `````````````````````````````````````
-
         Retrieve a list of dashboards that are associated with the given organization.
         If on the first page, this endpoint will also include any pre-built dashboards
```

Comment:
I don't quite understand this sentence `if on the first page`. Can you please expand on it?

Reply:
Hi, actually I think that sentence is no longer true since we've added dashboard templates to the product. I've removed this line! Also fixed some missing query parameters
<img width="628" alt="image" src="https://github.com/user-attachments/assets/3e8bd75f-4237-4537-bf95-42ff7fe4fcff">